news/2020-31-12-mappers-report-fall.md

```diff
@@ -0,0 +1,265 @@
+
+It's certainly well deserved, with each of their entries providing well built and interesting experiences that make excellent use of the challenges MBC provides. They are certainly a mapper to keep up with in the future as well!
+
+The MBC \#10 was also the last one in the year 2020. The contest will take a little break now, but fear not! We are sure that the next MBC won't take long once we are in 2021. So stay tuned and make sure to keep your eyes open for the news post!
+
+### osu! Mapping Olympiad #7: Results (osu!mania)
+
+The results of the osu! Mapping Olympiad are finally here and throwing a big surprise at us. For the first time in history of the Olympiad, two mappers share the 1st place with having the exact same score. Congratulation to [Kawawa](https://osu.ppy.sh/users/4647754) & [Toaph Daddy](https://osu.ppy.sh/users/7616811) being awareded for their outstanding performence with the tier 1 Mapping Olympiad Badge and the **Elite Mapper** user title!
+
+And as if this wouldn't be enough, the 3rd place is also shared by two mappers of the exactly same score. [lenpai](https://osu.ppy.sh/users/5314573) & [BringoBrango](https://osu.ppy.sh/users/10274043) managed as well to impress the judges with their entries. Both of their scores were pretty high and very close to the first ones, with only a difference of 2.5!
+
+We can't wait to see what the next osu! Mapping Olympiad will bring to us!
+
+### osu! Beatmapping World Championship 2020
+
+After 5 fast months, the osu! Beatmapping World Championship 2020 has come to an end. It's difficult to express just how amazing of a project and competition it was to watch, and each team brought their best selves - resulting in some of the closest rounds of the competition right at the end! For full details on the journey, scoring, and more, be sure to check out the [dedicated conclusion newspost](https://osu.ppy.sh/home/news/2020-10-22-beatmapping-world-championship-2020-concludes).
+
+The final results, in a simplified form, are as follows:
+
+#### Finals
+
+| Team 1 |  |  | Team 2 |
+| --: | :-: | :-: | :-- |
+| ![](/wiki/shared/flag/AR.gif) Argentina | 3 | **4** | **United Kingdom** ![](/wiki/shared/flag/GB.gif) |
+
+#### Third Place
+
+| Team 1 |  |  | Team 2 |
+| --: | :-: | :-: | :-- |
+| ![](/wiki/shared/flag/BR.gif) **Brazil** | **4** | 3 | Russian Federation ![](/wiki/shared/flag/RU.gif) |
+
+Congratulations to everyone!
+
+## <a id="around-community"></a>Around the Community
+
+Hello! We're back with the latest segment of Around the Community! This time we have two main spotlights: the return of the annual Christmas Queue, as well as a bunch of awesome guides that have been put out recently! Should you have something worth including that isn't here already, be sure to let [Feerum](https://osu.ppy.sh/users/4815717) or [Noffy](https://osu.ppy.sh/users/1541323) know about it! We're happy to give it more attention in our news post!
```

Comment:
you mention the christmas queue yet there's nothing written about it, and you don't mention the labour of love competition in this starter paragraph